apps/web/src/components/pages/HomePage/sections/Events.tsx

```diff
@@ -147,20 +149,29 @@ let filterArr: boolean[] = events.events.map(event => {
   return false;
 });
 
-const DEFAULT_EVENTS_TO_DISPLAY = 3;
-const CALENDAR_LINK =
+const FILTER_TYPES: string[] = [
+  "ACM Comp",
+  "ACM Data",
+  "ACM Game",
+  "ACM General",
+  "ACM Hack",
+  "ACM Sec",
+  "ACM-W"
+];
+const DEFAULT_EVENTS_TO_DISPLAY: number = 3;
+const CALENDAR_LINK: string =
   "https://calendar.google.com/calendar/embed?src=mst.edu_7u3stm8bn7l2umuastep5fmbl0%40group.calendar.google.com&ctz=America%2FChicago";
 
 const Events: React.FC<{}> = (): JSX.Element => {
-  const [filters, setFilters] = useState([filterArr]);
-  const [maxEvents, setMaxEvents] = useState(DEFAULT_EVENTS_TO_DISPLAY);
-  const [scrollYPosition, setScrollYPosition] = useState(0);
+  const [filters, setFilters] = useState<boolean[][]>([filterArr]);
```

Comment:
could this just be `boolean[]` and then you interact with it `filters[FILTER_TYPES.indefOx(group)]`

Reply:
So I tried that, but I was running into issues with stale props in the children (checkboxes) and they wouldn't update. Something to do with how functional components work and this was the only solution I could come up with.

Reply:
It because you have to update the entire list like
```
setFilters([filterArr]) 
```

because react state only does shallow equality check